docs/optimization_guide/dldt_optimization_guide.md

```diff
@@ -1,40 +1,40 @@
 # Introduction to Performance Optimization {#openvino_docs_optimization_guide_dldt_optimization_guide}
-Before exploring possible optimization techniques, let us first define what the inference performance is and how to measure that.
-Notice that reported inference performance often tends to focus on the speed of execution. 
-In fact these are at least four connected factors of accuracy, throughput/latency and efficiency. The rest of the document discusses how to balance these key factors. 
+In this guide You will be presented with information about definition of inference performance, how it can be measured, and what are the possible techniques to optimized it.
```

Comment:
```suggestion
This guide presents the definition of inference performance and how it can be measured. It also describes possible optimization techniques.
```